src/lib/relay/fetchQuery.ts

```diff
@@ -26,6 +26,7 @@ function _fetchQuery(queryID: string, variables: object, isQuery: boolean) {
   let body
   if (__DEV__) {
     body = { query: require("../../__generated__/complete.queryMap.json")[queryID], variables }
+    console.log(body)
```

Comment:
This turns out to be useful to debug graphQL queries when it doesn't work in emission for some reason

Reply:
What do you mean by “when it doesn't work in emission”?